Client/Frontend/Browser/HomePanel/favorites/FavoritesDataSource.swift

```diff
@@ -67,9 +67,10 @@ class FavoritesDataSource: NSObject, UICollectionViewDataSource {
         guard let fav = frc?.object(at: indexPath) else { return UICollectionViewCell() }
 
         cell.textLabel.text = fav.displayTitle ?? fav.url
-        cell.imageView.setIcon(fav.domain?.favicon, forURL: URL(string: fav.url ?? ""), scaledDefaultIconSize: CGSize(width: 40, height: 40), completed: { (color, url) in
+        let n: FaviconMO? = nil // Distinguishes which setIcon function is called
```

Comment:
This is kind of an interesting workaround, however, there should be a more determined solution than using a typed optional set to nil to choose the function. One option would be to give the `setIcon` function different declarations/names, to avoid overloading and calling the wrong function.